.github/pull_request_template.md

```diff
@@ -0,0 +1,19 @@
+<!---
+Please write your PR name in the present imperative tense. Examples of present imperative tense are:
+"Fix issue in the dispatcher where…", "Improve our handling of…", etc."
+
+For more information on Pull Requests, you can reference here:
+https://success.vanillaforums.com/kb/articles/228-using-pull-requests-to-contribute
+-->
+## Describe your changes
```

Comment:
Feel free to edit this as you want, but it sets a good baseline for pull requests.

Reply:
Thanks, added a checklist too.